libs/template/themes/design-tokens/src/storefront/index.ts

```diff
@@ -37,6 +38,7 @@ export const storefrontTokens: DesignToken[] = [
     },
     ...layoutMdTokens,
     ...typographyMediumTokens,
+    ...commonTokensSmall,
```

Comment:
`layoutMdTokens`, `typographyMediumTokens`, `commonTokensSmall`... I see two kind of inconsistency here:
- mixing Medium/Small and md/sm/lg
- different order like [domain][size]Tokens and [domain]Tokens[size]

Do we have any plans to make it similar in the future?

Reply:
Yes for sure. We like to use the same model but that is currently causing a cycle dep which why we postpone this for now